From 2d9e180858b3212507fcba7a7cb9d13f6a935a2f Mon Sep 17 00:00:00 2001
From: Lianbo Jiang <lijiang@redhat.com>
Date: Mon, 6 Jun 2022 19:09:16 +0800
Subject: [PATCH 10/89] Doc: update man page for the "bpf" and "sbitmapq"
 commands

The information of the "bpf" and "sbitmapq" commands is missing in the man
page of the crash utility.  Let's add it to the man page.

Signed-off-by: Lianbo Jiang <lijiang@redhat.com>
---
 crash.8 | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/crash.8 b/crash.8
index 1f3657b11e4c..e553a0b4adb3 100644
--- a/crash.8
+++ b/crash.8
@@ -584,6 +584,9 @@ creates a single-word alias for a command.
 .I ascii
 displays an ascii chart or translates a numeric value into its ascii components.
 .TP
+.I bpf
+provides information on currently-loaded eBPF programs and maps.
+.TP
 .I bt
 displays a task's kernel-stack backtrace.  If it is given the
 .I \-a
@@ -706,6 +709,11 @@ number of seconds between each command execution.
 .I runq
 displays the tasks on the run queue.
 .TP
+.I sbitmapq
+dumps the contents of the sbitmap_queue structure and the used
+bits in the bitmap. Also, it shows the dump of a structure array
+associated with the sbitmap_queue.
+.TP
 .I search
 searches a range of user or kernel memory space for given value.
 .TP
-- 
2.37.1

